### ISSUE TITLE
[Top Bar] Parent of nested sub-menu instantly navigates to its link when clicked and hover is disabled.

### ISSUE THREAD

**Issue by ZachWatkins** [zurb/foundation-sites#7468]
I am currently experiencing this issue in Foundation 5.5.3, importing the bower package. I believe this started in 5.2.0, as I have switched between many versions of 5.x trying to find one that works. This behavior does work as I expect it to in 5.0.2. This issue seems to be occurring because the "focus" event only triggers on "&lt;a&gt;" elements which have an "href" attribute, and "focus" resolves at the end of the "mousedown" event and therefore before the "click" event. The preventDefault behavior is in a click event on line 216 of foundation.topbar.js, and the focus event which limits preventDefault is on line 296.

How can we reproduce this bug?
1. Add an “href” attribute to the “&lt;a&gt;” element arranged before a dropdown “&lt;ul&gt;” element.
2. Click the “&lt;a&gt;” element.

What did you expect to happen?
After step two, the “hover” class would be applied to the parent of the “&lt;a&gt;” element. Clicking the “&lt;a&gt;” element a second time consecutively, and the page would navigate to the URL in its “href” attribute. 
In other words, how the first level of Top Bar's menu items behave when they have sub-menus.

What happened instead?
After step two, the page navigated to the URL in the “&lt;a&gt;” element’s “href” attribute.

Demonstration: http://codepen.io/zw/pen/pjmbEd?editors=100

Documentation from other users experiencing the same issue: http://foundation.zurb.com/forum/posts/26150-top-bar---problem-with-dropdown-when-clickable

**Comment by ZachWatkins** [comment]
I have a solution, but I do not know how to create this repository at version 5.5.4 locally. I cloned the repository but it is at the latest subversion of 6.

**Comment by ZachWatkins** [comment]
Please ignore the first two times I references this issue. I was having difficulty with revising the commit and decided to refork the repository instead.

**Comment by ZachWatkins** [comment]
I have submitted a pull request to address this issue: https://github.com/zurb/foundation-sites/pull/7572
